src/GridFieldOrderableRows.php

```diff
@@ -384,29 +383,25 @@ public function getManipulatedData(GridField $grid, SS_List $list)
                         $sortterm .= "$col $dir, ";
                     }
                 } else {
-                    $sortterm = $this->extraSortFields.', ';
+                    $sortterm = $this->extraSortFields . ', ';
                 }
             }
-            if ($list instanceof ArrayList) {
-                // Fix bug in 3.1.3+ where ArrayList doesn't account for quotes
-                $sortterm .= $this->getSortTable($list).'.'.$this->getSortField();
-            } else {
-                $sortterm .= '"'.$this->getSortTable($list).'"."'.$this->getSortField().'"';
-
-                if ($list instanceof DataList) {
-                    $classname = $list->dataClass();
-                    if ($defaultSort = Config::inst()->get($classname, 'default_sort')) {
-                        if (is_array($defaultSort)) {
-                            $defaultSortArray = [];
-                            foreach ($defaultSort as $column => $direction) {
-                                $defaultSortArray[] = "\"$column\" $direction";
-                            }
-                            $defaultSort = implode(', ', $defaultSortArray);
+
+            $sortterm .= '"' . $this->getSortField() . '"';
```

Comment:
Changing to v5 stuff the `$sortterm` var no longer requires the `getSortTable` call. That is unless I have misunderstood how this works (entirely possible). The tests were breaking without this change and now work if that helps